setup.py

```diff
@@ -43,6 +43,7 @@
         "sphinx-rtd-theme>=0.6",
         "sphinx-autodoc-typehints",
         "sphinx-click",
+        "typing_extensions",
```

Comment:
Local doc builds were passing, but I kept getting warnings that `WARNING: Failed guarded type import with ModuleNotFoundError("No module named 'typing_extensions'")`. I tried a few things w/ unpinning sphinx, themes, etc but nothing worked except just adding this to the actually install requirements. 🤷